src/main/java/pl/cyber/trainees/wyjasnienia/PracaDomowa.java

```diff
@@ -0,0 +1,49 @@
+package pl.cyber.trainees.wyjasnienia;
+
+public class PracaDomowa {
+
+    //Napisz metodę, która zwróci Twój aktualny wiek.
+    public void ileMaszLat(Integer wiek) {
+        System.out.println("Mój wiek to " + wiek);
+    }
+
+    // Napisz metodę, która zwróci Twoje imię.
+
+    public void mojeImie(String imie) {
+        System.out.println("Moje imię to " + imie);
+    }
+
+    // Napisz metodę, która jako argument przyjmuje liczbę i zwraca true jeśli liczba jest parzysta.
+
+    public boolean liczbaParzysta(Integer x) {
+        if (x%2==0) {
+            return true;
+        } else {
+            return false;
+        }
+
+    }
+
+    // 4. Napisz metodę, która jako argument przyjmuje liczbę
+    // i zwraca true jeśli liczba jest podzielna przez 3 i przez 5.
+
+    public boolean liczbaPodzielnaPrzezTrzy (Integer y) {
+        if (y%3==0 && y%5==0){
+            return true;
+        } else {
+            return false;
+        }
+    }
+    // 5. Napisz metodę, która jako argument przyjmuje liczbę i zwraca go podniesionego do 3 potęgi,
+    // podniesienie do potęgi realizujemy przez bibliotekę Math i jej metodę pow(liczba, doJakiejPotęgi)
+    // np. Math.pow(2,3)
+
+   // public static Integer potegowanie (Integer z){
+   //     Math.pow(z, 3);
```

Comment:
w dobrym kierunku Pani podążała.
Gdyby uprzeć się na metodę tylko pod potęgowanie wyglądałaby ona tak:
public static Double potegowanie (Double z, Double to){
     return Math.pow(z, to);
     }
     
     z racji tego, że jak spojrzymy na metodę pow on przyjmuje 2 double i zwraca double
     Natomiast rozwiązanie z SimpleApp też jest ok.
     W założeniu, że chcielibyśmy użyć wielokrotnie konstrukcji Math.pow() to utworzenie metody specjalizowanej pod to działanie jest troszkę lepsze.